src/pathways/escher.service.spec.ts

```diff
@@ -0,0 +1,90 @@
+// import {} from 'jasmine';
+
+import { EscherService } from './escher.service';
+
+describe('EscherService.alignReactions', () => {
+    let escherService: EscherService;
+
+    beforeEach(() => {
+        escherService = new EscherService();
```

Comment:
Any reason why you are not using dependency injection but are instead creating the service manually here? You could do something like this:
```ts
    beforeEach(inject((_escherService_) => {
        escherService = _escherService_;
    }));
```

This way you would also test that the service can be correctly injected (note that you would need to import the module in this case as you define all services there).